packages/dcos-integration-test/extra/test_composition.py

```diff
@@ -227,6 +237,8 @@ def test_systemd_units_are_healthy(dcos_api_session) -> None:
         role = node_health["Role"]  # Is one of master, agent, agent_public
         units_per_node[node] = set(node_health["Output"])
         exp_units_per_node[node] = expected_units[role]
+        if 'WinRM' in units_per_node[node]:
+            exp_units_per_node[node] = windows_units
```

Comment:
So this just verifies that there is a health report but not its state, correct? Should we also `assert 'WinRM' in units_per_node`?

Reply:
It checks state later here we only assign expected state to `windows_units` instead of `expected_units['agent']`